python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -125,6 +144,25 @@ def ReduceSumDimIntListEmptyDimModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ReduceSumDimIntListElementTypeBoolModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([1, 128], torch.bool, True),
+    ])
+    def forward(self, a):
+        return torch.sum(a, dim=(-1), keepdim=False).to(dtype=torch.bool)
+
+
+@register_test_case(module_factory=lambda: ReduceSumDimIntListElementTypeBoolModule())
+def ReduceSumDimIntListElementTypeBoolModule_basic(module, tu: TestUtils):
+    module.forward(tu.randint(1, 128).to(dtype=torch.bool))
```

Comment:
Can you set `high=2` in `tu.randint` to have a roughly even distribution of `True` and `False`? (`tu.randint` defaults to `high=10`, so this will produce many more `True`s than `False`)